spyder/utils/encoding.py

```diff
@@ -171,6 +171,13 @@ def encode(text, orig_coding):
     if orig_coding == 'utf-8-bom':
         return BOM_UTF8 + text.encode("utf-8"), 'utf-8-bom'
     
+    # Try saving with original encoding
+    if orig_coding:
+        try:
+            return text.encode(orig_coding), orig_coding
+        except UnicodeError:

```

Comment:
@rlaverde, I failed to see that in all other places we except in two errors: `UnicodeError` and `LookupError`. This shouldn't be an exception.

Please fix that in another PR.